cpp/src/io/orc/dict_enc.cu

```diff
@@ -205,47 +197,48 @@ CUDF_KERNEL void __launch_bounds__(block_size)
 {
   auto const col_idx    = blockIdx.x;
   auto const stripe_idx = blockIdx.y;
+  auto const t          = threadIdx.x;
   auto const& dict      = dictionaries[col_idx][stripe_idx];
   auto const& col       = columns[dict.column_idx];
 
   if (not dict.is_enabled) { return; }
 
-  auto const t         = threadIdx.x;
+  // Make a view of the hash map
+  auto const hash_fn        = hash_functor{col};
+  auto const equality_fn    = equality_functor{col};
+  using probing_scheme_type = cuco::linear_probing<map_cg_size, hash_functor>;
```

Comment:
nit: this has been declared twice. we could probably declare it right after `hash_functor`.

Reply:
All done!